src/easy3d.c

```diff
@@ -0,0 +1,203 @@
+#include <nitro.h>
+#include <string.h>
+#include <nnsys.h>
+
+#include "struct_defs/struct_0207C690.h"
+
+#include "unk_02017728.h"
+#include "easy3d.h"
+#include "gx_layers.h"
+#include "unk_02024220.h"
+
+void Easy3D_LoadModelFromResource(NNSG3dResMdl **outModel, NNSG3dResFileHeader **resource);
```

Comment:
**question:** Is this function not static? If it's not, then move its declaration to the header.